src/components/ebay-expand-button/component.js

```diff
@@ -0,0 +1,22 @@
+const assign = require('core-js-pure/features/object/assign');
+const eventUtils = require('../../common/event-utils');
+
+module.exports = {
+    onCreate(input) {
+        this.state = assign({}, input, {
```

Comment:
Since this is for marko4, we should not assign input to state for that. 
Since we are not changing any state variables like disabled, this should not be needed. Instead we can use `this.input.disabled` in the `handleClick` and `handleKeydown`